changelog/fix-8278-remove-redundant-text

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+Remove redundant message after account is onboarded
```

Comment:
```suggestion
Remove redundant message after the account is onboarded
```

Reply:
changed in https://github.com/Automattic/woocommerce-payments/pull/8399/commits/2adf682ee866d42705654cc9ae8b1e0678d6eb02 . thx.